### DISCUSSION TITLE
Board content does not reflect issue updates

### DISCUSSION THREAD

**Discussion by scolestock** [community/community#103186] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

The new github issue view in github projects is fantastic, but board content does not reflect issue updates right away, which can be pretty confusing. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Benedict-Forsberg** [comment]
I find that the content takes ages to update (even for simple things like moving a ticket between columns in the board view if editing the project custom fields instead of dragging the ticket across), and additional detail like pull requests are often omitted altogether.

Therefore I end up refreshing the page to pull the latest changes
